core/nodejsActionBase/runner.js

```diff
@@ -67,8 +67,7 @@ function NodeActionRunner() {
         } else {
             // The code is a plain old JS file.
             try {
-                eval(message.code);
-                thisRunner.userScriptMain = eval(message.main);
+                thisRunner.userScriptMain = eval('(function(){\n' + message.code + '\nreturn ' + message.main + '})()');
```

Comment:
Hum I think we don't want to continue to use eval.

I would preferred to write the  message.code to a file, and then have same flow as zip actions by doing a require on the file.
This means appending `exports.main = $message.main` to the file